GitUI/Translation/Russian.xlf

```diff
@@ -9256,6 +9256,11 @@ This will just checkout on the submodule the commit determined by the superproje
         <target>Не возможно подсветить выделенную ветку, когда граф ревизий загружается.</target>
         
       </trans-unit>
+      <trans-unit id="_baseForCompareNotSelectedError.Text">
+        <source>Base commit for compare does not selected.</source>
+        <target>Не выбран базовый коммит для сравнения.</target>
+
+      </trans-unit>
       <trans-unit id="_droppingFilesBlocked.Text">
         <source>For you own protection dropping more than 10 patch files at once is blocked!</source>
```

Comment:
Please remove this changes from PR. Translation should be done on https://www.transifex.com/git-extensions/git-extensions/